src/helpers/shifts.helper.ts

```diff
@@ -6,8 +6,20 @@ import { ColorHelper } from "./colors/color.helper";
 import { Colors } from "./colors/color.model";
 import { VerboseDateHelper } from "./verbose-date.helper";
 import { VerboseDate } from "../common-models/month-info.model";
+import { DataRow } from "../logic/schedule-logic/data-row";
 
 export class ShiftHelper {
+  public static getWorkersCount(day: number, data: DataRow[]): number {
```

Comment:
Przy okazji robienia TASK-66 okazało się, że w tym miejscu jako argument będziemy przekazywać  `ShiftInfoModel`, ponieważ tak jest wygodnie. 
Więc proszę zamienić to co jest teraz na funkcję która wygląda mniej więcej tak:
```typescript
function getWorkersCount(shifts: ShiftInfoModel) {
   const shiftsArray = Object.values(shifts)
 .........// liczenie 
}
```
Było by też fajnie, żeby ona zwracała listę, w której dla każdego dnia będzie podana liczba pracowników. Liczba dni === liczba kolumn w `Object.values(shifts)`